test/draw_line_string.test.js

```diff
@@ -137,7 +137,7 @@ test('draw_line_string render active line with 1 coordinate', t => {
     },
     geometry: {
       type: 'LineString',
-      coordinates: [[0, 0]]
+      coordinates: [[0, 0], [15, 15]]
```

Comment:
Yea, it was rendering an invalid LineString before. This happens in tests more than it does in real life as few people can click without moving the mouse and mousemove adds the second position.